android/app/src/main/cpp/app/organicmaps/Framework.cpp

```diff
@@ -1351,9 +1351,8 @@ JNIEXPORT void JNICALL
 Java_app_organicmaps_Framework_nativeSetRoutingListener(JNIEnv * env, jclass, jobject listener)
```

Comment:
Can it be null/unsubscribed? Or just set once? Can the listener address change?

Reply:
I don't understand a question. I did a minor style fix here without changing any logic.